examples/16-3-release-blog-create-ref.js

```diff
@@ -0,0 +1,19 @@
+class MyComponent extends React.Component {
+  // highlight-next-line
+  _divRef = React.createRef();
```

Comment:
Nit: we don’t use the underscore convention elsewhere in the docs.

Reply:
Gotcha. Long time habit.